src/test/java/com/jcabi/manifests/ManifestsTest.java

```diff
@@ -127,4 +130,70 @@ public void appendsAttributesFromInputStream() throws Exception {
         );
     }
 
+    /**
+     * When get on multiple manifests, the first element takes precedence.
```

Comment:
@pinaf Maintaining the order was appended in was not specified, but this was to establish the untested behaviour of the existing `get` method before I changed that piece of code.